.bazelrc

```diff
@@ -5,6 +5,8 @@
 build --conlyopt='-std=gnu11'
 build --cxxopt='-std=gnu++11'
 build --workspace_status_command=.circleci/workspace_status.sh
+build --stamp
+test --nostamp
```

Comment:
If this does not allow test caching, we could try setting `stamp` on targets of interest instead: https://docs.bazel.build/versions/master/user-manual.html#flag--stamp